org.mixedrealitytoolkit.spatialmanipulation/Solvers/SolverHandler.cs

```diff
@@ -351,6 +351,7 @@ protected virtual void Update()
         /// <summary>
         /// A Unity event function that is called every frame after normal update functions, if this object is enabled.
         /// </summary>
+        [System.Obsolete]
```

Comment:
nit: ` using System;` above  and `[Obsolete]` just for consistency.

Reply:
I agree, I missed it in the first editing, thanks for letting me know.  Fixed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/716/commits/865c7bff4727cb06d9c91f5dfe541340af63ac78 , resolving.